gradle.properties

```diff
@@ -21,3 +21,9 @@ kotlin.code.style=official
 licenseName="MIT License"
 licenseUrl=https://opensource.org/licenses/MIT
 scmUrl=https://github.com/rakutentech/android-miniapp.git
+
+#temp for POC Demo
+MINIAPP_SERVER_BASE_URL=https://miniapp-platform-japaneast-dev.azurewebsites.net/api/v1/
```

Comment:
Values for the base url, app version, id, and subscription key should not be committed, so please remove this section of `gradle.properties`.

We can set these values as environment variables on CircleCI. If you need it for testing locally, you can set it in your global gradle properites at `~/.gradle/gradle.properties`.